Makefile.overrides.mk

```diff
@@ -13,4 +13,4 @@
 # limitations under the License.
 
 # this repo is not yet on the container plan by default
-BUILD_WITH_CONTAINER ?= 0
+BUILD_WITH_CONTAINER ?= 1
```

Comment:
this will break a lot of the stuff relating to the prow cluster I think, as it won't mount the kubernetes context. Or those makefiles get the context on demand with gcloud tool which is not in the docker image.

It's possible the makefiles for prow stuff is completely separate and you need to be in the prow/ folder to run it and none of this applies

Reply:
This setting is only relevant for stuff you run from the command-line using make. Anything run from Bazel is unaffected, anything run form Prow is unaffected.

Reply:
I think there are one-off commands that are run though. under prow/makefile. It may not be included in the top level makefiles though, on which case this would not apply to it

Reply:
Yeah, I don't think that applies. This change only impacts build targets in Makefile.core.mk or anything it imports. Other makefiles are unrelated (although we'll eventually want to merge ther functionality as well so that all build activity can be automated and can be containerized)